styleguide/derek/pattern-components/service-level-teaser/_service-level-teaser.scss

```diff
@@ -0,0 +1,80 @@
+.serviceLevelTeaser {
+  background: $white;
+  border-bottom: 4px solid $brand-primary;
+  border-top: 4px solid $brand-primary;
+  box-shadow: 0 1px 3px $black-shadow;
+  display: block;
+  margin-bottom: 10px;
+  padding: 25px;
+  position: relative;
+  text-decoration: none;
+  width: 100%;
+
+  &:hover {
+    background-color: $gray-lightest;
+    text-decoration: none;
+  }
+
+  a {
+    @include standard-link;
+  }
```

Comment:
Insert typical comment about, is this nested anchor absolutely necessary here? Shouldn't we just be targetting `.serviceLevelTeaser-bottomLink`, or am I misunderstanding the use case?